### PR TITLE
Initial packaging of asciinema

### PR DESCRIPTION
**Pull request by ranok**

### CODE REVIEW

File: projects/asciinema.org/asciinema/package.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+build:
+  script: python-venv.sh {{prefix}}/bin/asciinema
+
+provides:
+  - bin/asciinema
+
+test:
+  dependencies:
+    python.org: ^3.12
```
Comment (jhheider) [inline projects/asciinema.org/asciinema/package.yml]: tests will bring in runtime dependencies; so this shouldn't be needed here.

**Review by jhheider — CHANGES_REQUESTED** [review]
package name should just be `asciinema.org`

thanks for the contribution!

**Comment by ranok** [comment]
> package name should just be `asciinema.org`
> 
> thanks for the contribution!

I am going to package agg, which is by the same author, but converts the asciinema files into animated GIFs, I was thinking I'd put it under the same asciinema.org parent, but it's a different binary (Rust vs python)

File: projects/asciinema.org/asciinema/package.yml (hunk 1 above)
Comment (ranok) [reply projects/asciinema.org/asciinema/package.yml]: Fixed, thanks

**Comment by jhheider** [comment]
> I am going to package agg, which is by the same author, but converts the asciinema files into animated GIFs, I was thinking I'd put it under the same asciinema.org parent, but it's a different binary (Rust vs python)

that should be fine. there's no requirement that a node not be both a namespace and a package. see: `rust-lang.org` and `rust-lang.org/cargo`. i think it's intuitive that the package with the namespace's name should be named that way.

**Comment by ranok** [comment]
> > I am going to package agg, which is by the same author, but converts the asciinema files into animated GIFs, I was thinking I'd put it under the same asciinema.org parent, but it's a different binary (Rust vs python)
> 
> that should be fine. there's no requirement that a node not be both a namespace and a package. see: `rust-lang.org` and `rust-lang.org/cargo`. i think it's intuitive that the package with the namespace's name should be named that way.

Ok, I will put agg as a sub-dir in the parent, and I moved this one up

**Review by jhheider — APPROVED** [review]

**Pull request merged**